multiapps-controller-core/src/main/java/org/cloudfoundry/multiapps/controller/core/validators/parameters/RoutesValidator.java

```diff
@@ -45,9 +51,17 @@ public boolean isValid(Object routes, final Map<String, Object> context) {
         return true;
     }
 
-    private boolean isElementUnsupportedOrInvalid(Entry<String, Object> entry) {
-        ParameterValidator validator = validators.get(entry.getKey());
-        return validator == null || !validator.isValid(entry.getValue(), null);
+    private boolean isElementUnsupportedOrInvalid(Entry<String, Object> entry, final Map<String, Object> context) {
```

Comment:
I think context should be renamed to routesElement because this is a private function dedicated only for this use case where we pass route entry and routesElement. However, this is my personal perception becai

Reply:
Changed the names, the previous ones were too generic, I agree